helper/tests/codegen-111/src/main/compat-avro/under111/DeeplyNested.avsc

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
is this schema used at all?

Reply:
yes. it generates classes Outer Middle and Inner later used in the test (to deserialize into)